test/e2e/argo_server_test.go

```diff
@@ -1572,6 +1572,43 @@ func (s *ArgoServerSuite) TestRetryStoppedButIncompleteWorkflow() {
 	})
 }
 
+func (s *ArgoServerSuite) TestRetryWorkflowWithContinueOn() {
+	var workflowName string
+	s.Given().
+		Workflow(`@testdata/retry-workflow-with-continueon.yaml`).
+		When().
+		SubmitWorkflow().
+		WaitForWorkflow(fixtures.ToBeFailed).
+		Then().
+		ExpectWorkflow(func(t *testing.T, metadata *metav1.ObjectMeta, status *wfv1.WorkflowStatus) {
+			workflowName = metadata.Name
```

Comment:
shall we validate the node count here as well, so we know retry produces an identical result?

Reply:
OK